### DISCUSSION TITLE
有可能出一款简配的中枢网关吗？

### DISCUSSION THREAD

**Discussion by Jimmo-o** [XiaoMi/ha_xiaomi_home#104] [General]
有可能出一款简配的中枢网关吗？不需要极客版，只需要让设备接入home assistant本地化，可以降低网关性能和定价

**Comment by huhuime** [comment]
路由器里带的就是减配，而且有人已经在试图提取docker了

**Reply by chemwolf6922** [reply]
路由器里带的是完整的，不是减配

**Reply by huhuime** [reply]
硬件层面减配，比如存储空间更小

**Reply by chemwolf6922** [reply]
> 硬件层面减配，比如存储空间更小

4G的emmc已经是最小的型号了（现在4G的都不好找）。Emmc砍到nand/nor flash不只是空间减小。。读写性能（尤其是随机）也会下降一大截。
同理512M更小的RAM就得去考虑集成RAM的SOC了，性能会掉一大截（像绿米全系的ssd201那种）。

**Reply by ooodwbooo** [reply]
提取的docker镜像和路由器有交互，在别的环境里启动不起来

**Reply by MrDotW** [reply]
> 提取的docker镜像和路由器有交互，在别的环境里启动不起来

是不是后台还有其它的docker 类似haos一堆服务才能运行

**Comment by erchuyi** [comment]
有必要吗？也就300块钱一个

**Reply by Jimmo-o** [reply]
内置中枢的路由器价格和其他品牌的路由器差不多，但小米路由器风评又不太好

**Reply by jjcszxh** [reply]
> 内置中枢的路由器价格和其他品牌的路由器差不多，但小米路由器风评又不太好，小米路由器风评完全是自己不会优化而已

**Comment by LOVE2CMOL** [comment]
你是否在寻找 小米多模2 ？

**Reply by Jimmo-o** [reply]
多模还要找旧固件

**Reply by vincywindy** [reply]
多模要旧固件是因为官方封堵了验证，那只要官方想，直接集成就不存在固件问题。

**Reply by Jimmo-o** [reply]
> 多模要旧固件是因为官方封堵了验证，那只要官方想，直接集成就不存在固件问题。

应该不会适配了吧

**Comment by Hugh696** [comment]
be3600pro，600多已经很简配了，而且中枢网关300不到的价格也没什么降价空间了

**Comment by chemwolf6922** [comment]
极客版才吃几兆内存，裁了又能如何呢😂

**Reply by Jimmo-o** [reply]
你是极客版的开发？中枢网关性能是不是特够用😂

**Comment by MrGuoRanDuo** [comment]
如果能直接挂在电脑或者开发板上就好了  家里有电脑是7x24不关机的

**Comment by kulongwangzhi85** [comment]
我倒是希望米家集成能利用ha内部集成的mqtt,完成消息订阅功能。这不就是简化版的中枢
